app/src/main/java/org/kontalk/util/MessageUtils.java

```diff
@@ -708,17 +735,29 @@ public static String toString(byte[] text) {
         return text;
     }
 
-    public static void convertSmileys(Editable input) {
+    public static boolean convertSmileys(Editable input) {
```

Comment:
Yes, it's called very often. convertSmiley() is called every time the text in the input field changes. [called within afterTextChanged()]